parm/config/config.anal

```diff
@@ -62,81 +62,116 @@ export OBERROR=$FIXgsi/prepobs_errtable.global
-#   identical to ../global_convinfo.txt.  Thus, the logic below is not
-#   needed at this time.
-#   Assimilate COSMIC-2 GPS
-#   if [[ "$CDATE" -ge "2020052612" && "$CDATE" -lt "YYYYMMDDHH" ]]; then
-#     export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2020052612
-#   fi
-
-
-#   Turn off assmilation of OMPS during period of bad data
-    if [[ "$CDATE" -ge "2020011600" && "$CDATE" -lt "2020011806" ]]; then
-	export OZINFO=$FIXgsi/gfsv16_historical/global_ozinfo.txt.2020011600
-    fi
-
-
-#   Set satinfo for start of GFS v16 parallels
-    if [[ "$CDATE" -ge "2019021900" && "$CDATE" -lt "2019110706" ]]; then
-	export SATINFO=$FIXgsi/gfsv16_historical/global_satinfo.txt.2019021900
-    fi
-
-#   Turn on assimilation of Metop-C AMSUA and MHS
-    if [[ "$CDATE" -ge "2019110706" && "$CDATE" -lt "2020022012" ]]; then
-	export SATINFO=$FIXgsi/gfsv16_historical/global_satinfo.txt.2019110706
-    fi
-
-#   NOTE:  
-#   As of 2020022012, gfsv16_historical/global_satinfo.txt.2020022012 is
-#   identical to ../global_satinfo.txt.  Thus, the logic below is not
-#   needed at this time
-#
-#   Turn off assmilation of all Metop-A MHS
-#   if [[ "$CDATE" -ge "2020022012" && "$CDATE" -lt "YYYYMMDDHH" ]]; then
-#       export SATINFO=$FIXgsi/gfsv16_historical/global_satinfo.txt.2020022012
-#   fi
+	export ABIBF="/dev/null"
+	if [[ "$CDATE" -ge "2019022800" ]] ; then
+		export ABIBF="$DMPDIR/${CDUMP}x.${PDY}/${cyc}/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
+		if [[ "$CDATE" -ge "2019111000" && "$CDATE" -le "2020052612" ]]; then
+			export ABIBF="$DMPDIR/${CDUMP}y.${PDY}/${cyc}/${CDUMP}.t${cyc}z.gsrcsr.tm00.bufr_d"
+		fi
+	fi
+
+	export AHIBF="/dev/null"
+	if [[ "$CDATE" -ge "2019042300" ]]; then
+		export AHIBF="$DMPDIR/${CDUMP}x.${PDY}/${cyc}/${CDUMP}.t${cyc}z.ahicsr.tm00.bufr_d"
+	fi
+
+	export HDOB=$DMPDIR/${CDUMP}x.${PDY}/${cyc}/${CDUMP}.t${cyc}z.hdob.tm00.bufr_d
+
+	#   Use dumps from NCO GFS v16 parallel
+	if [[ "$CDATE" -ge "2020103012" ]]; then
+		export ABIBF=""
+		export AHIBF=""
+		export HDOB=""
+	fi
+
+	#   Set info files and prepobs.errtable.global for GFS v16 retrospective parallels
+	if [[ "$CDATE" -ge "2019021900" && "$CDATE" -lt "2019110706" ]]; then
+		export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2019021900
+		export OBERROR=$FIXgsi/gfsv16_historical/prepobs_errtable.global.2019021900
+	fi
+
+	#   Place GOES-15 AMVs in monitor, assimilate GOES-17 AMVs, assimilate KOMPSAT-5 gps
+	if [[ "$CDATE" -ge "2019110706" && "$CDATE" -lt "2020040718" ]]; then
+		export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2019110706
+		export OBERROR=$FIXgsi/gfsv16_historical/prepobs_errtable.global.2019110706
+	fi
+
+	#   Assimilate 135 (T) & 235 (uv) Canadian AMDAR observations
+	if [[ "$CDATE" -ge "2020040718" && "$CDATE" -lt "2020052612" ]]; then
+		export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2020040718
+		export OBERROR=$FIXgsi/gfsv16_historical/prepobs_errtable.global.202004071
```

Comment:
The date got cut off at the end of this file.  It should be 2020040718

Reply:
Good catch. Fixed.